solidity/dashboard/src/components/Modal.jsx

```diff
@@ -89,6 +171,7 @@ export const ModalContextProvider = ({ children }) => {
     }
     setModalOptions(null)
     setIsOpen(false)
+    // dispatch({ type: "modal/is_closed" })
```

Comment:
Let's remove this if it is unnecessary.

Reply:
Fixed in 6497b6cca5aaab8475f0604c5d1670f3cef5f9d5